web_app_drawer/static/src/xml/web_app_drawer.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<templates>
+
+	<t t-name="web_app_drawer.SearchField">
+		<form class="form-inline app-drawer-search-form">
+			<div class="form-group">
+				<label class="sr-only" for="appDrawerSearch">
+					Search
+				</label>
+				<input type="text"
+					   class="form-control pull-right app-drawer-search-input"
+					   id="appDrawerSearch"
+					   placeholder="Search"
+					   />
```

Comment:
Bah I did that so long ago, but it seems that maybe something is horribly borked in the new version that came out the other day. Stupid Komodo!

Seriously though, I'll figure out what the heck is up with my settings - this is annoying. Spaces4Life!